### PR TITLE
Adding some Codechef problems and solutions

### PR DESCRIPTION
**Pull request by deepak1214**
Hey there!, Thanks for submitting a PR to this repo :smiley:  
Please complete the following checklist for your PR to be approved. Mark the items by adding an `x`.   
Example:  - [ ] Mark me as completed  
To mark: - [x] Mark me as completed

- [x] My Pull Request adheres to the **Contribution guidelines**.
- [x] I have added `problem.md` and `solution` files in the **appropriate folder**.
- [x] In `problem.md` I have provided a **correct link** to the problem.  
  (In case you are not adding a problem, you can freely mark this as complete)
- [x] The solution I have given is my **own** and not plagiarized in any form.
- [x] My solution passes all the testcases within the given constraints.
- [x] I have properly indented and **documented my solution using comments**.
- [x] *I agree that all the above marked are true to the best of my knowledge and my PR can be closed if found otherwise.*

If any of the above is unmarked, please make the necessary changes and then mark them so that you're PR can be approved and merged.  
(Only completed checklist PRs will be reviewed)

### CODE REVIEW

**Comment by deepak1214** [comment]
Sorry for the "awkward" title in the pull request.

File: CodeChef_problems/AND Plus OR/solution.cpp [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#include<bits/stdc++.h>
```
Comment (Vishruth-S) [inline CodeChef_problems/AND Plus OR/solution.cpp:1]: Please check your solution again.
For the input 
```
2
1
8
```
Your code outputs
```
0 1
0 8
```
Expected correct output:
```
0 1
5 3
```

**Review by Vishruth-S — CHANGES_REQUESTED** [review]

File: CodeChef_problems/AND Plus OR/solution.cpp (hunk 1 above)
Comment (deepak1214) [reply CodeChef_problems/AND Plus OR/solution.cpp:1]: Sir, I have already mentioned in the comments that the problem belongs to constructive algorithm , and there are many possible solution of the single problem, you could even check my solution correctness by submitting it in Codechef .

- [ ]

**Comment by deepak1214** [comment]
Sir, I have already mentioned in the comments of the solution that the problem belongs to the Constructive Algorithm section ,
and there are many solutions of the same problem can be possible. You can even check my code's correctness here:-

image

**Review by Vishruth-S — APPROVED** [review]
@deepak1214  Yeah your solution was correct after all, it was some bug with Codechef IDE 😅 . Great job 👍 
Sorry for the mistake from my end, and thanks for contributing 😄

**Comment by deepak1214** [comment]
It's alright  sir, Thanks 😃

**Pull request merged**